scripts/send-email-to-pre-fxa-subscribers.js

```diff
@@ -8,42 +10,52 @@ const EmailHelpers = require("../template-helpers/emails.js");
 const EmailUtils = require("../email-utils");
 const { LocaleUtils } = require ("../locale-utils");
 
+const PAGE_SIZE = process.env.PAGE_SIZE;
```

Comment:
Should this have a default (ie: `process.env.PAGE_SIZE || 100` -- or whatever @Micheletto thinks is a sane default)?
I can't imagine OPs would try running this without that env var set, but I can think this might fail spectacularly below if it tries to use an `undefined` `perPage` page size.

Reply:
Oh dang, good point. Running without a `PAGE_SIZE` will try to run without any pagination. So it'll just run/fail the same as it does now. But I'll make the point on the bug.